gulp/docs.js

```diff
@@ -45,20 +45,31 @@ module.exports = {
 				if (!isInComponents) { return acc; }
 
 				var componentName = extractComponentName(file);
-				var docs = reactDocgen.parse(fs.readFileSync(file), function (ast, recast) {
-					var definition;
-					recast.visit(ast, {
-						visitObjectExpression: function (path) {
-							_.forEach(path.get('properties').value, function (property) {
-								if (property.key.name === 'render') {
-									definition = path;
-								}
-							});
-							return false;
-						}
-					});
-					return definition;
-				});
+				var docs = reactDocgen.parse(
```

Comment:
Totally, that's what I was thinking. I started going down that rabbit hole but decided to save it for another day!